packages/runtime-security/zarf.yaml

```diff
@@ -14,11 +14,17 @@ metadata:
     dev.uds.title: UDS Core (Runtime Security)
     dev.uds.tagline: Enable more advanced security with Runtime inspection
     dev.uds.category: UDS-Core,Runtime Security
-    dev.uds.keywords: uds,neuvector
+    dev.uds.keywords: uds,neuvector,falco
 
 components:
   # Neuvector
   - name: neuvector
     required: true
     import:
       path: ../../src/neuvector
+
+  # Falco
+  - name: falco
+    required: false # Optional component until we swap out Neuvector
+    import:
+      path: ../../src/falco
```

Comment:
I think we'll also want to import the falco validate/e2e into [this task file](https://github.com/defenseunicorns/uds-core/blob/1709_falco-zarf-package/packages/runtime-security/tasks.yaml) to make sure they run on single layer tests.